### ISSUE TITLE
how to diagnose errors

### ISSUE THREAD

**Issue by samashley** [greghendershott/frog#134]
First, thanks for Frog!! I just discovered it. Very nice!

Also I'm new to github so forgive me if this is not the way to ask this but:

When trying to build a blog if something goes wrong where can one get info about the errors that frog/racket encountered???

**Comment by greghendershott** [comment]
Just a quick reply from phone: Try the --verbose or --very-verbose command
line flags. Does that help? If not, what sort of errors did you mean?

**Comment by samashley** [comment]
Thanks for your answer! And I'm a bit embarrassed because I might have thought of that, sorry. But yes, I'll try increasing the verbosity.

Basically what I meant was that on my first try I modified a lot of things throughout the example that was generated with --init but when I tried to build it all I got was a generic web page about the racket server. I must have had a typo in one of the templates, or some other change that frog/racket didn't like. So I wondered if there might be a detailed log somewhere with the errors that were encountered. After asking I of course realized that those must be reported in the terminal.

In any case thanks again for this nice program!! I've only just begun to use it but everything about using it seems to me to be *much* more clear than with the others that I've come across.
